django_app/redbox_app/redbox_core/models.py

```diff
@@ -46,6 +46,11 @@ def sanitise_string(string: str | None) -> str | None:
 
 
 class AISettings(UUIDPrimaryKeyBase, TimeStampedModel):
+    class ChatBackend(models.TextChoices):
+        GPT_35_TURBO = "gpt-35-turbo-16k", _("gpt-35-turbo-16k")
+        GPT_4_TURBO = "gpt-4-turbo-2024-04-09", _("gpt-4-turbo-2024-04-09")
+        GPT_35_OMNI = "gpt-4o", _("gpt-4o")
```

Comment:
This Enum value for GPT-4o looks wrong

Reply:
good spot!

Reply:
[ee26cea](https://github.com/i-dot-ai/redbox/pull/958/commits/ee26ceaa8596cf8b6314a03d1babdf0853190f8b)